src/tram-space/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./tram-space')
```

Comment:
this kinda syntax is not contributing to a folder/ file clutter is it?
Recently I was in the camp of "there is no such thing as too many folders" and now I feel like i'm double backing a lot on that, well names files (that organize themselves alphabetically) and low folder count makes it a lot easier to manage and take in.

Reply:
My biggest goal is to keep test and source files organized. The `index.js` just makes it so imports are easier.

I'm not adverse to a different pattern if you have an example / suggestion.

I agree that some of the folders make it... Difficult.

Reply:
Since test files are (and will continue to be) in a separate folder, I may remove the folders and have a flatter structure.